recipes/pyomo.extras/meta.yaml

```diff
@@ -0,0 +1,46 @@
+# Do metapackages have a version?
+# Do metapackages have a license?
+
+package:
+  name: pyomo.extras
+  version: 3.0
+
+build:
+  noarch_python: True
+  number: 0
+
+requirements:
+  run:
+    - python
+    - ipython
+    - openpyxl
+    - sympy
+    - pymysql
+    - pyodbc
+    - pyro4
+    - pyyaml
+    - suds-jurko
+    - xlrd
+
+test:
+  imports:
+    - IPython
+    - openpyxl
+    - sympy
+    - pymysql
+    - pyodbc
+    - Pyro4
```

Comment:
Oh sorry. I was thinking these were run deps for a second. Not Python imports. Please forgive the noise.